Fix luhn's checksum wrong test values and its implementation.

oalbe opened the pull request:
I finally managed to do it. Issue #284 pointed out the wrong checksum values in the test cases for the `luhn` exercise. I corrected them and corrected the implementation.

There's something I have to point out regarding the implementation of the exercise (`example.py`) though. I didn't really correct it, I just replaced the old one with my implementation _slightly_ modified. It works and all tests pass, but it would be nice to first give it a look and highlight potential problems.

As for the _slightly_ differences with my implementation all I changed is:
- added the `(object)` part to the first line `class Luhn(object)`. I don't really have a precise idea what it does, but it was there in the old implementation so I added it.
- removed the comments.
- cleaned up the unnecessary white spaces from empty lines (since last time I remember Flake8 complaining).

I hope I did everything the right way.

behrtam commented:
Why not just move the `% 10` to the appropriate position? That small change would sufficiently satisfy the changed test cases.

```python
def checksum(self):
    return sum(self.addends())

def is_valid(self):
    return self.checksum() % 10 == 0
```

oalbe commented:
Because... I didn't think about it. I took back the old implementation and made the fix. I also added the empty line at the end of the file and rebased into a single commit. Should be fine now.

behrtam commented:
You accidentally changed the import in the test file with the new commit.

```
 -from luhn import Luhn
 +from example import Luhn
```

Please shorten your commit message which also shouldn't end with a point ([more about commit msg](http://chris.beams.io/posts/git-commit/#seven-rules)). Than we are good to go.

oalbe commented:
Done, I hope in a good way. And rebased everything in a single commit once again. (it's quite a challenge to fit the description for something in just 50 characters)

behrtam commented:
Thanks for finding and fixing it.

oalbe commented:
Thanks to you for the help.

kytrinyx commented:
Sweet, I appreciate the fix!
